myriad-scheduler/src/main/java/org/apache/myriad/configuration/MyriadConfiguration.java

```diff
@@ -81,10 +82,14 @@
 
   public static final String DEFAULT_NATIVE_LIBRARY = "/usr/local/lib/libmesos.so";
 
-  public static final Integer DEFAULT_ZK_TIMEOUT = 20000;
+  public static final Integer DEFAULT_ZK_TIMEOUT    = 20000;
 
   public static final Integer DEFAULT_REST_API_PORT = 8192;
-
+  
+  public static final String DEFAULT_ROLE           = "*";
+  
+  public static final String DEFAULT_ZK_SERVERS     = "localhost:2181";
+  
```

Comment:
Gotcha, will update accordingly

On Wed, May 11, 2016 at 10:41 PM, Darin <notifications@github.com> wrote:

> In
> myriad-scheduler/src/main/java/org/apache/myriad/configuration/MyriadConfiguration.java
> <https://github.com/apache/incubator-myriad/pull/71#discussion_r62957848>:
>
> >
> >    public static final Integer DEFAULT_REST_API_PORT = 8192;
> > -
> > +
> > +  public static final String DEFAULT_ROLE           = "*";
> > +
> > +  public static final String DEFAULT_ZK_SERVERS     = "localhost:2181";
> > +
>
> Let's not reformat to align = signs. We're not doing it elsewhere so the
> code get's inconsistent.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-myriad/pull/71/files/9bb1e6b7514d58b5cb128bd3e23bec0a03f55aaa#r62957848>
>